src/ipa_pc/mod.rs

```diff
@@ -736,6 +736,155 @@ impl<G: AffineCurve, D: Digest> PolynomialCommitment<G::ScalarField> for InnerPr
         })
     }
 
+    /// Refers to the "Multi-poly multi-point assertions" section of the "Recursive SNARKs based on Marlin" document
+    fn batch_open_individual_opening_challenges<'a>(
```

Comment:
I'd propose to rebrand that function into something like "batch_open",  and comment that it does a multi-point (multi poly) batching as from Boneh et al.

Reply:
Actually this is an internal function. The function which called from the outside code when this library is used is named "batch_open"

Reply:
"batch_open" takes a single opening challenge while "batch_open_individual_opening_challenges" takes a function pointer.
This is a further abstraction introduced by arkworks, but we can remove it when we are going to move to sponge based function.
I suggest to leave it as it is for now, to avoid too much code refactoring (that we will do it anyway in the future)